src/diffusers/models/upsampling.py

```diff
@@ -17,11 +17,15 @@
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+from packaging import version
 
 from ..utils import deprecate
 from .normalization import RMSNorm
 
 
+is_torch_less_than_2_1 = version.parse(version.parse(torch.__version__).base_version) < version.parse("2.1")
```

Comment:
We already have these utilities here:
https://github.com/huggingface/diffusers/blob/14a1b86fc7de53ff1dbf803f616cbb16ad530e45/src/diffusers/utils/import_utils.py#L689

Reply:
Resolved in latest commit